### PR TITLE
Us 248

### PR DESCRIPTION
**Pull request by amansharma96**
### Required* What is the context of this pull request? Why is it being done?
- [x ] Bug fix
- [ x] New feature
  - [x ] UI change
  - [ ] Logic change (Briefly describe the change below)
  - 
- [x ] Refactor
- [ ] Other (please specify below)

### Required* before your pull request can be merged
- [ ] All relevant tests pass
- [ ] I code reviewed my own code and made sure it was cleaned up and annotated

### Optional
- [ ] I wrote and/or updated relevant tests
- [ ] My pull request is less than 400 lines of code

### Optional - Additional Information
* **Add screenshots here for any UI changes (try to limit to the most important changes to avoid extending the page too much).**

* **Are there any related PRs? Link to them here.**

* **Link to any additional documentation (Design doc, wireframes, etc.)** **required when available*

### CODE REVIEW

**Review by clhughe9 — DISMISSED** [review]
A couple things that aren't quite matching MYN/CERT pages yet:
- when I first open the Hazard page, if i click next immediately, it takes me to the next tab without prompting me that I havent entered valid info for lat/long or the type of hazard
- There custom date/time picker is not set which allows for setting/changing of the date time for the report
- the "Notes" section on the second tab is required but should be optional

**Review by Sheldon-Pang — DISMISSED** [review]
Need to merge from Dev

**Comment by clhughe9** [comment]
mine says branch US-248 is up to date with develop

File: EmergencyDataCollectionApp/src/utils/Database/OfflineSQLiteDB.js [hunk 1]
```diff
@@ -272,3 +274,31 @@ export function truncateTable(callback) {
     },
   );
 }
+
+
+
+export const fetchHazardReports = (callback) => {
```
Comment (Sheldon-Pang) [inline EmergencyDataCollectionApp/src/utils/Database/OfflineSQLiteDB.js:280]: You don't need a separate database setup; we are using a single database. Use the queryReportsByType(reportType, setReports) function to fetch the report.

**Comment by clhughe9** [comment]
Hey Aman one more thing, the cancel button doesn't match the other pages cancel buttons, but that's the last thing I can see. nice work.

**Review by dchanley — APPROVED** [review]
Works as intended

**Pull request merged**